tests/lib/mockMagic.py

```diff
@@ -9,3 +10,73 @@ def validate_config_for_device(self, device_mac, config):
             return return_validate_config_for_device['return_validate_intf']
         if config == validate_true['config']:
             return return_validate_config_for_device['return_validate_true']
+
+    def device_decommissioning(self, device_id, request_id):
+        """
+        mock method for cvprac device_decommissioning()
+        """
+        if device_id == device_data[0]["serialNumber"]:
+            self.result = {'value': {'key': {'requestId': request_id},
+                                     'deviceId': device_id},
+                           'time': '2022-02-12T02:58:30.765459650Z'}
+        else:
+            self.result = None
+
+    def device_decommissioning_status_get_one(self, request_id):
+        """
+        mock method for cvprac device_decommissioning_status_get_one()
```

Comment:
add that the expectation is that the self.result is set by `device_decommissioning` when called first